afs/templates/views/components/reports/observation.htm

```diff
@@ -58,16 +64,7 @@ <h1 class="afs-report-toolbar-title"><span class="afs-report-name">{% trans "Obs
                     }
                 }"></div>
             </div>            
-            <!-- Location Tab -->
-            <div class="afs-report-page" data-bind="if: activeSection() === 'location'">
```

Comment:
"Location" tab is removed by the ticket but it is still a visible node. I think it should appear somewhere. The description may be the place to go?